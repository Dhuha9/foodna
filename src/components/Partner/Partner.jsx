import React, { Component } from 'react';

export default class Partner extends Component {
  render() {
    return (
      <div className="w-75 mx-auto">
        <img className="partner-image" src={this.props.img} alt="sponsor" />
      </div>
    );
  }
}
