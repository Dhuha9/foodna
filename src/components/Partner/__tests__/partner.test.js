import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Partner from '../Partner';

it('renders apply now correctly', () => {
  const tree = renderer.create(<Partner />).toJSON();
  expect(tree).toMatchSnapshot();
});
