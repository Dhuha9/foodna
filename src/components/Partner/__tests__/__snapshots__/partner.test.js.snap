// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apply now correctly 1`] = `
<div
  className="w-75 mx-auto"
>
  <img
    alt="sponsor"
    className="partner-image"
  />
</div>
`;
